source/fuzz/fuzzer_pass_interchange_zero_like_constants.cpp

```diff
@@ -85,6 +85,11 @@ void FuzzerPassInterchangeZeroLikeConstants::Apply() {
     uint32_t constant_id = constant->result_id();
     uint32_t toggled_id = FindOrCreateToggledConstant(constant);
 
+    if (GetTransformationContext()->GetFactManager()->IdIsIrrelevant(constant_id) ||
+        GetTransformationContext()->GetFactManager()->IdIsIrrelevant(toggled_id)) {
+      continue;
+    }
+
```

Comment:
Perhaps we should check whether constant_id is irrelevant immediately after getting its result id and continue if so.

Then we could have FindOrCreateToggledConstant assert that it only gets applied to a non-irrelevant id, and to be sure to only return a non-irrelevant constant.

Would that be reasonable to do in this PR?

Reply:
I think so, I'll fix that

Reply:
Fixed in 35f5730.